cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreComputedValueUpdateReference.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion.update;
+
+import java.util.function.Function;
+
+import com.powsybl.iidm.network.Identifiable;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ */
+public class TripleStoreComputedValueUpdateReference extends TripleStoreSimpleUpdateReference {
+
+    public TripleStoreComputedValueUpdateReference(String predicate, String contextReference, Function<Identifiable, String> valueComputation) {
+        super(predicate, contextReference);
+        this.valueComputation = valueComputation;
```

Comment:
Assert that valueComputation are not null?

Reply:
Done